Packs/CommonScripts/Scripts/HttpV2/HttpV2.yml

```diff
@@ -0,0 +1,204 @@
+  type: textArea
+  isArray: false
+  name: body
+  required: false
+  secret: false
+- default: false
+  description: |-
+    Specify the Content-Type header for the request.
+    Shorthands are provided for the following common content types:
+    json (application/json)
+    xml (text/xml)
+    form (application/x-www-form-urlencoded)
+    data (multipart/form-data)
+    If you choose to define a different type, please include the full type name, e.g: application/pdf
+  isArray: false
+  name: request_content_type
+  required: false
+  secret: false
+- default: false
+  description: |-
+    Specify the Accept header for the request - the response content type.
+    Shorthands are provided for the following common content types:
+    json (application/json)
+    xml (text/xml)
+    form (application/x-www-form-urlencoded)
+    data (multipart/form-data)
+    If you choose to define a different type, please include the full type name, e.g: application/pdf
+  isArray: false
+  name: response_content_type
+  required: false
+  secret: false
+- auto: PREDEFINED
+  default: false
+  defaultValue: raw_response
+  description: Specify how you would like to parse the response.
+  isArray: false
+  name: parse_response_as
+  predefined:
+  - text
+  - json
+  - xml
+  - raw_response
+  required: false
+  secret: false
+- default: false
+  description: 'Basic authorization. Please set values in the format: username,password.
+    For Bearer token please use the headers.'
+  isArray: false
+  name: auth_credentials
+  required: false
+  secret: true
+- default: false
+  description: URL parameters to specify the query.
+  type: keyValue
+  isArray: false
+  name: params
+  required: false
+  secret: false
+- default: false
+  defaultValue: '10'
+  description: Specify the timeout of the HTTP request in seconds. Defaults to 10
+    seconds.
+  isArray: false
+  name: timeout
+  required: false
+  secret: false
+- auto: PREDEFINED
+  default: false
+  defaultValue: 'True'
+  description: The request will be called again with the new URL.
+  isArray: false
+  name: enable_redirect
+  predefined:
+  - 'True'
+  - 'False'
+  required: false
+  secret: false
+- default: false
+  description: ' Specify the array of status codes that should cause a retry. For example: 301-303,400,402.'
+  isArray: false
+  name: retry_on_status
+  required: false
+  secret: false
+- default: false
+  defaultValue: '3'
+  description: How many retries should be made in case of a failure.
+  isArray: false
+  name: retry_count
+  required: false
+  secret: false
+- default: false
+  defaultValue: '5'
+  description: Specify the timeout between each retry in seconds. Defaults to 5.
+  isArray: false
+  name: timeout_between_retries
+  required: false
+  secret: false
+- auto: PREDEFINED
+  default: false
+  defaultValue: 'no'
+  description: Save the response in a file.
+  isArray: false
+  name: save_as_file
+  predefined:
+  - 'yes'
+  - 'no'
```

Comment:
change to boolean

Reply:
We kept it as yes and no cause that is the way it is used in the V1 script.